oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/SearchAppLayout.jsx

```diff
@@ -23,14 +23,17 @@ export const ActiveFiltersCountFloatingLabelComponent = ({
   className,
 }) => {
   const { filters } = currentQueryState;
-  const searchAppContext = useContext(SearchConfigurationContext);
-  const {
-    initialQueryState: { filters: initialFilters },
-  } = searchAppContext;
+  const { aggs } = useContext(SearchConfigurationContext);
```

Comment:
Would be nice it this could be hook `useActiveFilters()` as this pattern already repeats in few places